### ISSUE TITLE
"Play Multiplayer" instead of "Play Onlline"

### ISSUE THREAD

**Issue by farooghkz** [minetest/minetest#5982]
As I would type 192.168.1.104 in the address textbox(I mean in a LAN) so not always "Play Online"

**Comment by SmallJoker** [comment]
As soon you're playing in a network you're online, just limited to a subnet of the whole internet. Thus, connecting to a server in your network is "Online", as other people can join the server aswell if they're in the same network.

**Comment by Wuzzy2** [comment]
I think you need to look up “online” in the dictionary:
https://en.wiktionary.org/wiki/online#English

I am neutral on this issue, both wordings are correct.

**Comment by octacian** [comment]
Well, "Play Multiplayer" honestly looks and sounds nicer, however, it is longer. I think that "Play Online" is fine for now personally. It is technically correct, though I can see that some *could* misunderstand it.

Also neutral on this.

**Comment by DS-Minetest** [comment]
What about making it to only "Multiplayer" without the "Play "?

**Comment by red-001** [comment]
The `play` in `play muliplayer` is redundant. But it I still think that `Play Online` or `Online` is better since you are playing multiplayer when you host a server.

**Comment by rubenwardy** [comment]
I prefer `Start` and `Join`.
Or `Host` and `Join`, but slightly more technical

**Comment by red-001** [comment]
`Host` and `Join` sound good

**Comment by farooghkz** [comment]
@SmallJoker  @Wuzzy2, You are right but a lot of people don't know this.

**Comment by red-001** [comment]
If we are going to get that technical then singleplayer is also online.

**Comment by TekhnaeRaav** [comment]
"Multiplayer" sounds to much like it could be local, while "online",  I think informs you enough that it is multiplayer, because you don't really play online alone. At least in most persons perspective.

I would also like to suggest "Online Game" for the button to keep it consistent to "Local Game".

**Comment by paramat** [comment]
I don't like 'online', it somehow seems cheesy. Technically yes LAN is online but the common understanding of online is internet. 'Join' seems better to me.

**Comment by rubenwardy** [comment]
Should be start and join imo

**Comment by paramat** [comment]
'Host' seems better to me.

**Comment by Wuzzy2** [comment]
Lol, you're seriously still debating this after 5 months? Just decide for one wording and get done with it. Or just reject the issue and close it. No need to discuss this to death.

Jesus Christ …

**Comment by Ezhh** [comment]
I have to agree with Wuzzy. Either decide to change, or not to change and then close the issue. Something as simple as a wording choice shouldn't need half a year to resolve.

**Comment by paramat** [comment]
Well i'm fine with 'Start' and 'Join' so will make a PR.
('start' and 'join' instead of 'local game' and 'play online' that is).

**Comment by ThomasMonroe314** [comment]
Maybe "Singleplayer" would work better instead of "Start Game" ?

**Comment by paramat** [comment]
Thanks for reminding me, now doing.

**Comment by paramat** [comment]
PR #6956

**Comment by ThomasMonroe314** [comment]
lool #6955 is mine

**Comment by farooqkz** [comment]
Shouldn't someone close this?
